docs/proposals/mcs/remove-derived-mcs.md

```diff
@@ -0,0 +1,352 @@
+make
+```
+
+After the compilation is successful, the corresponding binary will be generated in the root directory.
+
+Mirror making and installation or upgrading please refer to [coreDNS official website](https://coredns.io/).
+
+###### Configuration instructions
+
+The suffix `clusterset.local` is used as an example to illustrate
+
+* Configure coredns permissions in member2
+
+`kubectl edit clusterrole system:coredns`
+
+```shell
+...
+- apiGroups:
+   - multicluster.x-k8s.io
+   resources:
+   - serviceimports
+   verbs:
+   - list
+   - watch
+```
+
+* Configure multicluster zone rules
+
+Add `multicluster` processing rules to corefile in `coredns`
+
+```shell
+  kubectl edit configmap coredns -n kube-system
+  ....
+    Corefile: |
+     .:53 {
+         errors
+         health {
+            lameduck 5s
+         }
+         ready
+         multicluster clusterset.local # Add this line.
+         kubernetes cluster.local in-addr.arpa ip6.arpa {
+            pods insecure
+            fallthrough in-addr.arpa ip6.arpa
+            ttl 30
+         }
+         prometheus:9153
+         forward ./etc/resolv.conf {
+            max_concurrent 1000
+         }
+         cache 30
+         the loop
+         reload
+         load balance
+     }
+...
+```
+
+* Configure dns search item "clusterset.local"
+
+**Configure node /etc/resolv.conf file**
+
+Edit the `/etc/resolv.conf` file and add the "clusterset.local" entry to give priority to local services when using the domain name `foo.default.svc`.
+
+```shell
+root@btg:/# cat /etc/resolv.conf
+search default.svc.cluster.local svc.cluster.local cluster.local clusterset.local
+nameserver 10.13.0.10
+options ndots: 5
+```
+
+This configuration method needs to configure the `/etc/resolv.conf` file on the client node. If you don't want to change the node's `resolv.conf` file, you can also use the following method.
+
+**Configure pod's dnsConfig in yaml**
+
+That is, add the configuration item searches of dnsConfig in yaml, and set the value to `clusterset.local`, as shown below:
+
+```yaml
+...
+       dnsPolicy: ClusterFirst
+       dnsConfig:
+         searches:
+           -clusterset.local
+         options:
+           - name: timeout
+             value: "2"
+```
+
+###### Access method
+
+Since the search function of dns is used, the `foo` service  access method is:
+
+1. `foo.default.svc` access. This method gives priority to local access, and if the local service with the same name is unavailable, it will access the remote cluster service.
+
+2. `foo.default.svc.clusterset.local` full domain name access. The request will go directly to the remote server for processing.
+
+###### Advantages and disadvantages
+
+shortcoming:
+
+1. Using `foo.default.svc` access cannot achieve load balancing. The remote service will only be accessed when the local service is unavailable. Therefore, if you need to achieve local and remote access load balancing, you need to cooperate with other solutions.
```

Comment:
We can achieve load balancing by changing the DNS resolve configuration.

Reply:
> We can achieve load balancing by changing the DNS resolve configuration.

Sorry， I mean load balancing for local and remote services. 
not  for the local services, right?